lang/funcs/facts/core/load_posix.go

```diff
@@ -0,0 +1,39 @@
+// Mgmt
+// Copyright (C) 2013-2018+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// +build !darwin
+
+package core // TODO: should this be in its own individual package?
+
+import (
+	"syscall"
+)
+
+// load returns the system load averages for the last minute, five minutes and
+// fifteen minutes. Calling this more often than once every five seconds seems
+// to be unnecessary, since the kernel only updates these values that often.
+// TODO: is the kernel update interval configurable?
+func load() (one, five, fifteen float64, err error) {
+	var sysinfo syscall.Sysinfo_t
+	if err = syscall.Sysinfo(&sysinfo); err != nil {
+		return
+	}
+	one = float64(sysinfo.Loads[0]) / LoadScale
```

Comment:
Shouldn't the `LoadScale` const get moved here too?

Reply:
Correct, but in all the excitement about the other stuff I forgot to do the thing I set out for. I think the method I implemented for darwin also works on other osses, and now with the test suite and docker going I can test it. I'll verify if it works and push a universal solution again if it does.

Reply:
@aequitas Great! However, please keep the pure-golang version in for Linux, in case we ever hit any reasons why we would need that.